libmattermost.h

```diff
@@ -35,6 +35,8 @@
 #define MATTERMOST_USER_PAGE_SIZE 200 // 200 is MAX. in paged queries (and default)
 #define MATTERMOST_HISTORY_PAGE_SIZE 60 // 200 is MAX in paged queries (60 is default)
 #define MATTERMOST_MAX_PAGES 10 // that is 2000 users or posts in paged queries
+#define MATTERMOST_NEW_CHANNEL_FOUND -2 // -2 indicates that timestamp does not exist for a new channel
+#define MATTERMOST_NO_OF_CHANNEL_HISTORY_MESSAGES 60 // fetch last 60 messages for a new channel
```

Comment:
Why aren't you using the existing variable `MATTERMOST_HISTORY_PAGE_SIZE`?

Reply:
there is no any existing constant for timestamp, however moved it now onto a separate line

Reply:
I was not talking about the timestamp in this comment. There is a variable called `MATTERMOST_HISTORY_PAGE_SIZE` on line no. 36 here. Why didn't you use that instead of creating a new variable `MATTERMOST_NO_OF_CHANNEL_HISTORY_MESSAGES`?

Reply:
I think you should use the existing variable for now and we'll create a new variable in the future if we need to increment this value.